Correct description of transform-origin

brady-ds opened the pull request:
* **Problem:** The description of `transform-origin` should explain that, effectively, the transform origin is translated to (0, 0), the CSS `transform` property is applied so that those transformations are about the transform origin, and then the transform origin is translated back to its original location.  Instead, the description says the reverse: (0, 0) is translated to the transform origin, the `transform` property is applied, and then (0, 0) is restored back to its original location.  This would incorrectly make the transformations apply around the point mirroring the transform origin.  The code example immediately following also contradicts the wrong description.

* **URL:** <https://developer.mozilla.org/en-US/docs/Web/CSS/transform-origin>

* **Issue:** None

brady-ds committed:
Correct description of transform-origin's equivalent in transformations so that those transformations are given in the correct order.

ericwbailey commented on files/en-us/web/css/transform-origin/index.html:
```diff
@@ -19,8 +19,7 @@
 
-<p>This property is applied by first translating the element by the value of the property, then applying the element's transform, then translating by the negated property value.<br>
- This means, this definition</p>
+<p>This property is applied by first translating the element by the negated value of the property, then applying the element's transform, then translating by the property value. This means, this definition</p>
```
I don't think "by the negated value of the property" is clear enough. Can we rephrase it to be more descriptive, like in your PR description?

brady-ds committed:
Expand on the description of `transform-origin` to explain the reasoning behind the translations.

brady-ds committed:
Normalize "transformation origin" to "transform origin" in the prose to match the CSS property name.

brady-ds commented:
Besides expanding the indicated paragraph as requested, I also added [b0382b2](https://github.com/mdn/content/pull/4925/commits/b0382b29c62f5470af7d52b5934071d99a68c298) to clean up language in the prior paragraph that was inconsistent with the property name and therefore inconsistent with the text I added.  Since it could be considered an unrelated correction, please let me know if you would prefer it as a separate PR.

ericwbailey commented on files/en-us/web/css/transform-origin/index.html:
```diff
@@ -31,6 +30,8 @@
 <pre class="brush: css">transform-origin: 0 0;
 transform: translate(-100%, 50%) rotate(45deg) translate(100%, -50%);</pre>
 
+<p>where, reading from right to left, <code>translate(100%, -50%)</code> is the translation to bring the transform origin to the true origin, <code>rotate(45deg)</code> is the original transformation, and <code>translate(-100%, 50%)</code> is the translation to restore the transform origin to its original location.</p>
```
```suggestion
<p>Reading from right to left, <code>translate(100%, -50%)</code> is the translation to bring the transform origin to the true origin, <code>rotate(45deg)</code> is the original transformation, and <code>translate(-100%, 50%)</code> is the translation to restore the transform origin to its original location.</p>
```

ericwbailey reviewed (changes requested):
This is great, thank you. One last little tweak, and then I think this is good to go!

brady-ds committed:
For readability, break a long sentence at a code snippet.

Co-authored-by: Eric Bailey <ericwbailey@users.noreply.github.com>

ericwbailey reviewed (approved):
Wonderful. Thank you for your effort on this, and apologies for the delay in getting back to you. Also, congrats on your first MDN contribution!
